### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Comment by SuperQ** [comment]
What is the `config.yml` in the project root? It looks like a CircleCI config.

**Comment by roidelapluie** [comment]
> What is the `config.yml` in the project root? It looks like a CircleCI config.

leftover from my work, sorry :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  tests:
+    jobs:
+    # Support the last two go relases, as per https://golang.org/dl/
+    - test:
+        name: go-1-14
+        go_version: "1.14"
+    - test:
+        name: go-1-15
+        go_version: "1.15"
+        # Style is only checked against the latest supported Go version.
```
Comment (SuperQ) [inline .circleci/config.yml:52]: I know this is mostly a copy-n-paste from client_golang, but I think we can improve this.

If we split style out as a separate job, we can avoid this. Take a look at [gosnmp CircleCI](https://github.com/gosnmp/gosnmp/blob/c615d988dbf658fcafd2236faa199d624782ce7f/.circleci/config.yml#L53-L67) for an example.

**Comment by roidelapluie** [comment]
@SuperQ done

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**